SafeNeighbourhood/Pages/Index.cshtml.cs

```diff
@@ -15,11 +16,22 @@ public void OnGet()
         {
             using (var webClient = new WebClient())
             {
+                //Get Summary Neighborhood Cleanliness Data
+                string neighborhoodData = webClient.DownloadString("https://data.cincinnati-oh.gov/resource/h8mv-4fsc.json");
+                List<NeighborhoodData> neighborhoods = NeighborhoodData.FromJson(neighborhoodData);
+
+                //Get Vacant Properties in Cincinnati
                 string propertyData = webClient.DownloadString("https://data.cincinnati-oh.gov/resource/w3jp-dfxy.json");
+                IEnumerable<PropertyData> properties = PropertyData.FromJson(propertyData);
+
+                //Filter Vacant Properties by truly Vacant Properties
+                IEnumerable<PropertyData> vacantProperties = properties.Where(property => property.DataStatusDisplay.Contains("Vacant"));
```

Comment:
Filtering properties to only those with "Vacant" in the DataStatusDisplay field because the rest of the properties returned by the Json are not really vacant unless this field contains the word "Vacant" in it.